library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -15,20 +15,19 @@
     /**
      * Constant for use with {@link #orderBy}
      */
-    static final int ORDER_ASCENDING = 1;
+    public static final int ORDER_ASCENDING = 1;
```

Comment:
yes we want clients using the library to be able to call this method
 Add a line note